ibm_catalog.json

```diff
@@ -187,9 +190,13 @@
             {
               "name": "deploy-arch-ibm-account-infra-base",
               "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
-              "flavors": ["resource-group-only"],
+              "flavors": [
+                "resource-group-only"
+              ],
               "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
-              "ignore_auto_referencing": ["*"],
+              "ignore_auto_referencing": [
```

Comment:
This is not required. Asked [here](https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1747395625959889)